frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -109,19 +167,27 @@ export const sessionsTableLogic = kea<sessionsTableLogicType<Moment, SessionType
         },
     }),
     actionToUrl: ({ values }) => ({
-        setFilters: () => {
-            return buildURL(values.selectedDateURLparam)
-        },
+        setFilters: () => buildURL(values.selectedDateURLparam, values.sessionRecordingId),
+        loadSessionPlayer: () => buildURL(values.selectedDateURLparam, values.sessionRecordingId),
+        closeSessionPlayer: () => buildURL(values.selectedDateURLparam, values.sessionRecordingId),
```

Comment:
This + urlToAction feels like it violates DRY a lot, plus this logic is getting huge.

Is there a pattern here that would help clean things up?